db/index.js

```diff
@@ -0,0 +1,434 @@
+    }
+    line[j] = line[j].replaceAll("'", "''");
+  }
+
+  querySkusString += `('${line[0]}', '${line[1]}', '${line[2]}', '${line[3]}'), `;
+  countSkus += 1;
+  if (countSkus === 1000) {
+    querySkusString = querySkusString.substring(0, querySkusString.length - 2);
+
+    pool.query(
+      `INSERT INTO skus(sku_id, style_id, size, quantity) VALUES ${querySkusString}`,
+      (skusErr, skusResult) => {
+        if (skusErr) {
+          console.error('Error executing query', skusErr.stack);
+        }
+        return skusResult;
+      }
+    );
+    querySkusString = '';
+    countSkus = 0;
+  }
+};
+skusRl.on('close', () => {
+  const extraSkusQuery = querySkusString.substring(0, querySkusString.length - 2);
+  pool.query(
+    `INSERT INTO skus(sku_id, style_id, size, quantity) VALUES ${extraSkusQuery}`,
+    (err, result) => {
+      if (err) {
+        console.error('Error executing extra query', err.stack);
+      }
+
+      return result;
+    }
+  );
+});
+
+// Photo CSV Handler
+
+const photosFilename = './server/photos.csv';
+
+const photosReader = fs.createReadStream(photosFilename);
+const photosRl = readline.createInterface({
+  input: photosReader,
+  crlfdelay: Infinity,
+});
+photosRl.on('line', (line) => {
+  handlePhotosLine(line);
+});
+let queryPhotosString = '';
+let countPhotos = 0;
+
+const queryPhotosObj = {};
+let handlePhotosLine = (line) => {
+  line = line.split(/,(?=(?:(?:[^"]*"){2})*[^"]*$)/);
+  for (let j = 0; j < line.length; j += 1) {
+    if (line[j][0] === '"') {
+      line[j] = line[j].substr(1);
+    }
+    if (line[j][line[j].length - 1] === '"') {
+      line[j] = line[j].substr(0, line[j].length - 1);
+    }
+    line[j] = line[j].replaceAll("'", "''");
+  }
+
+  if (queryPhotosObj[line[0]] === undefined) {
+    queryPhotosString += `('${line[0]}', '${line[1]}', '${line[2]}', '${line[3]}'), `;
+    countPhotos += 1;
+    queryPhotosObj[line[0]] = 1;
+  }
+
+  if (countPhotos === 1000) {
+    queryPhotosString = queryPhotosString.substring(
+      0,
+      queryPhotosString.length - 2
+    );
+
+    pool.query(
+      `INSERT INTO photos(id, style_id, url, thumbnail_url) VALUES ${queryPhotosString}`,
+      (photoErr, photoResult) => {
+        if (photoErr) {
+          console.error('Error executing first photos query', photoErr.stack);
+        }
+        return photoResult;
+      }
+    );
+    queryPhotosString = '';
+    countPhotos = 0;
+  }
+};
+photosRl.on('close', () => {
+  const extraPhotosQuery = queryPhotosString.substring(0, queryPhotosString.length - 2);
+  if (extraPhotosQuery.length > 0) {
+    pool.query(
+      `INSERT INTO photos(id, style_id, url, thumbnail_url) VALUES ${extraPhotosQuery}`,
+      (err, result) => {
+        if (err) {
+          console.error('Error executing extra photos query', err.stack);
+        }
+
+        return result;
+      }
+    );
+  }
+});
+
+// Styles CSV Handler
+
+const stylesFilename = './server/styles.csv';
```

Comment:
If we had more time, it would be cool to refactor this to cut down on repeated code. This looks solid though!